examples/analog_hamiltonian_simulation/09_Noisy_quantum_dynamics_for_Rydberg_atom_arrays.ipynb

```diff
@@ -0,0 +1,1691 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.        found_key = modify_dict(performance.__dict__)
</pre><p>You could get a dict from <code>performance.dict()</code> rather than modifying the internals of the pydantic object </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
If we use `performance.dict()`, we will need to reconstruct the `Performance` object from the dictionary, and I don't know how to do that easily because `Performance` has other subfields. Instead, I found it easier to just modify the internal dict object of `performance` in place and recursively.